install/kubernetes/helm/istio/charts/gateways/values.yaml

```diff
@@ -18,6 +18,14 @@ istio-ingressgateway:
     # SDS server that watches kubernetes secrets and provisions credentials to ingress gateway.
     # This server runs in the same pod as ingress gateway.
     image: node-agent-k8s
+    resources:
```

Comment:
Just curious, what would happen without these resource requests?

Reply:
horizontal pod autoscalar requires `requests` on all containers, it does not work otherwise.